src/Servers/Kestrel/test/InMemory.FunctionalTests/RequestTests.cs

```diff
@@ -1048,27 +1048,52 @@ public async Task ContentLengthReadAsyncPipeReaderBufferRequestBodyMultipleTimes
         }
 
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/23628")]
```

Comment:
I think this is a clear enough fix that we can remove the QuarantinedTest attribute immediately, but I can leave it quarantined and let it become unquarantined naturally.